docs/docs/getting-started/quickstart.md

```diff
@@ -75,7 +75,7 @@ Available options:
                            yourself.
   --persistence-dir DIR    The directory where the Hydra Head state is stored.Do
                            not edit these files manually!
-  --network-id INTEGER     Network identifier for a testnet to connect to. We
+  --testnet-magic INTEGER     Network identifier for a testnet to connect to. We
```

Comment:
I would recommend to re-generate this output using `hydra-node --help` to be sure you have the right indentation.